utils/unittest/execution_state.go

```diff
@@ -19,12 +19,12 @@ import (
 const ServiceAccountPrivateKeyHex = "e3a08ae3d0461cfed6d6f49bfc25fa899351c39d1bd21fdba8c87595b6c49bb4cc430201"
 
 // Pre-calculated state commitment with root account with the above private key
-const GenesisStateCommitmentHex = "6cd39d62143db2407fef486d109188bb52a2ab647e48685a98990a7aaa775833"
+const GenesisStateCommitmentHex = "739e0b30cdef7da507e0ad11e65dc6bd06c866c63ba31499bb63fbd4611fb44a"
```

Comment:
Wondering how this is calculated? 🤔

Reply:
When we run the test and it fails, the output tells us what the expected root hash is (it actually computes it from the root state). So when the test fails, we just copy the expected hash here.